src/main/java/uk/gov/companieshouse/search/api/interceptor/UserAuthorisationInterceptor.java

```diff
@@ -36,7 +36,7 @@ private boolean validateAPI(HttpServletRequest request, HttpServletResponse resp
         DataMap.Builder builder = new DataMap.Builder()
                 .requestId(request.getHeader(REQUEST_ID_HEADER_NAME));
 
-        if(AuthorisationUtil.hasInternalUserRole(request) && PUT.matches(request.getMethod())) {
+        if(AuthorisationUtil.hasInternalUserRole(request) && !GET.matches(request.getMethod())) {
```

Comment:
Will this block a health check request that will normally be a `GET` request?

Reply:
I think adding some unit tests for GET requests without different types of authentication would help

Reply:
GET requests unit tests added in latest commit
@johncook-ch as the healthcheck route is not added to the registry, it wouldn't be affected.